packages/bot-web-ui/src/components/quick-strategy.jsx

```diff
@@ -62,77 +64,141 @@ const QuickStrategyForm = ({
+                                                        )
+                                                    }
+                                                />
+                                            ) : (
+                                                <Dropdown
+                                                    placeholder={localize('Trade type')}
+                                                    is_align_text_left
+                                                    list={trade_type_dropdown}
+                                                    name='trade_type'
+                                                    value={selected_trade_type.value}
+                                                    onChange={e => {
+                                                        onChangeDropdownItem(
+                                                            'trade-type',
+                                                            e.target.value,
+                                                            setFieldValue
+                                                        );
+                                                    }}
+                                                />
+                                            )}
+                                        </>
                                     )}
                                 </Field>
                             </div>
                             <div className='quick-strategy__form-row quick-strategy__form-row--multiple'>
                                 <Field name='quick-strategy__duration-unit'>
                                     {({ field }) => (
-                                        <Autocomplete
-                                            {...field}
-                                            autoComplete='off'
-                                            className='quick-strategy__duration-dropdown'
-                                            type='text'
-                                            label={localize('Duration unit')}
-                                            list_items={duration_unit_dropdown}
-                                            disabled={duration_unit_dropdown.length === 1}
-                                            onHideDropdownList={() => {
-                                                onHideDropdownList('duration-unit', values[field.name], setFieldValue);
-                                            }}
-                                            onItemSelection={({ value }) => {
-                                                onChangeDropdownItem('duration-unit', value, setFieldValue);
-                                            }}
-                                            onScrollStop={() => onScrollStopDropdownList('duration-unit')}
-                                        />
+                                        <>
+                                            {!is_mobile ? (
```

Comment:
it's clear if you revert this condition 
is_mobile ? (<Dropdown../> : <Autocomplete.../>